frontend/projects/abtesting/src/app/shared/components/delete/delete.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { MatDialogRef } from '@angular/material';
+
+@Component({
+  selector: 'app-delete',
+  templateUrl: './delete.component.html',
+  styleUrls: ['./delete.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DeleteComponent {
+
+  deleteConfirm: string;
+  message: boolean;
```

Comment:
@YashilDepani Can we update this variable name to more descriptive? may be  isDeleteButtonClicked?

Reply:
code updated